releases/xnat/charts/xnat-web/templates/configmap.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "xnat-web.fullname" . }}
+  labels:
+    {{- include "xnat-web.labels" . | nindent 4 }}
+data:
+  passwd: |
+    root:x:0:0:root:/root:/bin/bash
+    daemon:x:1:1:daemon:/usr/sbin:/usr/sbin/nologin
+    bin:x:2:2:bin:/bin:/usr/sbin/nologin
+    sys:x:3:3:sys:/dev:/usr/sbin/nologin
+    sync:x:4:65534:sync:/bin:/bin/sync
+    games:x:5:60:games:/usr/games:/usr/sbin/nologin
+    man:x:6:12:man:/var/cache/man:/usr/sbin/nologin
+    lp:x:7:7:lp:/var/spool/lpd:/usr/sbin/nologin
+    mail:x:8:8:mail:/var/mail:/usr/sbin/nologin
+    news:x:9:9:news:/var/spool/news:/usr/sbin/nologin
+    uucp:x:10:10:uucp:/var/spool/uucp:/usr/sbin/nologin
+    proxy:x:13:13:proxy:/bin:/usr/sbin/nologin
+    www-data:x:33:33:www-data:/var/www:/usr/sbin/nologin
+    backup:x:34:34:backup:/var/backups:/usr/sbin/nologin
+    list:x:38:38:Mailing List Manager:/var/list:/usr/sbin/nologin
+    irc:x:39:39:ircd:/var/run/ircd:/usr/sbin/nologin
+    gnats:x:41:41:Gnats Bug-Reporting System (admin):/var/lib/gnats:/usr/sbin/nologin
+    nobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin
+    _apt:x:100:65534::/nonexistent:/usr/sbin/nologin
+    tomcat:x:{{ default "1000" .Values.securityContext.runAsUser }}:{{ default "1000" .Values.securityContext.runAsUser }}::/data/xnat/home:/usr/sbin/nologin
```

Comment:
I *really* dislike this, but I don't see another way to do it without container rebuilds.

(Btw are you handling `/etc/group` as well?)

Reply:
I was having issue with permissions and suspected that tomcat/xnat required a home folder location set for some functionality. This can stand some further testing.
I was not intending to add the group file as this would only be required for name look up on things like directory listings, as it stands the group id number is displayed and while this may cause some confusion I was going to leave it in favour one less thing to configure.

Reply:
If the password file is required for home folder setting it can not be done within the container. This is due to two considerations, the first being that any id can be selected for the deployment and if this was being set up on OpenShift a random user id would be created by default during provisioning. The second is that the container is to eventually become read only as a security measure so the password file would not be able to be updated from within the container.